Script.js

```diff
@@ -1,18 +1,218 @@
+        </p>
+        <p>
+          <img src="images/Counter.png" alt="">                            
+        </p>
+        <p>
+          <span class="date">${workProperties[project].year}</span>
+        </p>
+      </div>
+    </div>
+    <p class="work_summary">${workProperties[project].summary}</p>
+    <ul class="tags">    
+    </ul>
+    <div class="project_link btn_effect">
+        See Project
+    </div>
+  </div> 
+  </div>
+  `;
+  workSection.appendChild(workItem);
+  addTechnologies(project);
+}
+
+//  Iterate through work properties and add elements and project property values to the work section
+Object.keys(workProperties).forEach((property) => addProject(property));
+
+//  Popup window variables
+const overlay = document.createElement('section');
+overlay.classList.add('popupOverlay');
+overlay.classList.add('hide');
+const projectWindow = document.createElement('div');
+projectWindow.classList.add('hide');
+projectWindow.classList.add('popup');
+
+function popupWindowClasses() {
+  projectWindow.classList.toggle('hide');
+  overlay.classList.toggle('hide');
+  body.classList.toggle('noscroll');
+}
+
+//  Close window function
+function closePopupWindow() {
+  const closePopup = document.querySelector('.popupClose');
+  closePopup.addEventListener('click', () => {
+    popupWindowClasses();
+  });
+}
+
+//  Add popup window html
+function popupWindow(id) {
+  popupWindowClasses();
+  body.appendChild(overlay);
+  body.appendChild(projectWindow);
+  projectWindow.innerHTML = `  
+  <div class="work_details">
+    <div class="project">
+      <h2 class="work_item_header">${workProperties[id].name}</h2>
+      <button class='popupClose'>&times</button>
+      <div class="work_history">
+        <p class="client">
+          <span class="client_name">${workProperties[id].client}</span>
+        </p>
+        <p>
+          <img src="images/Counter.png" alt="">                            
+        </p>
+        <p class="role">
+          <span class="role_name">${workProperties[id].role}</span>
+        </p>
+        <p>
+          <img src="images/Counter.png" alt="">                            
+        </p>
+        <p>
+          <span class="date">${workProperties[id].year}</span>
+        </p>
+      </div>
+    </div>
+    <img src="${workProperties[id].featuredImage}" alt="project snapshot" class="portifolio_img">
+    <p class="work_summary">${workProperties[id].description}</p>
+    <ul class="tags">
+    </ul>
+    <img src="Images/Separator Bottom.png" class='popupSeperator' alt="">
+    <div class="popupLinks">
+      <div class="project_link btn_effect">
+        <p>
+          <a href="#">See Live</a>
+          <img src="Images/External link icon.png" alt="">
+        </p>
+      </div>
+      <div class="project_link btn_effect">
+        <p>
+          <a href="#">See Source</a>
+          <img src="Images/Github popup icon.png" alt="">
+        </p>
+      </div>
```

Comment:
- kindly make sure the icons for `see live and see source` are showing correctly as expected, I suggest you rename correctly the images, use the right path and resize them as it shows in the Figma design  Kindly make sure your design stick as much as possible to the Figma design en required in the project  :+1: :+1: 

## You design 

![image](https://user-images.githubusercontent.com/95297251/202664695-59ad4f54-5ea8-45a3-b871-39fa990552b6.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/202664603-9863638c-1ae7-4108-93c0-a691b1307c3b.png)